google-api-client/src/test/java/com/google/api/client/googleapis/mtls/MtlsTransportBaseTest.java

```diff
@@ -35,11 +35,11 @@ protected KeyStore createTestMtlsKeyStore() throws IOException, GeneralSecurityE
     return SecurityUtils.createMtlsKeyStore(certAndKey);
   }
 
-  protected static class TestMtlsProvider implements MtlsProvider {
-    private boolean useClientCertificate;
-    private KeyStore keyStore;
-    private String keyStorePassword;
-    private boolean throwExceptionForGetKeyStore;
+    public static class TestMtlsProvider implements MtlsProvider {
```

Comment:
Please run a google style formatter on the source files you've touched. Bonus points for enabling lint checks on the new apache-v5 module (alternative: please create an issue in this repo to add a lint check)